test/Libraries/RevitIntegrationTests/RevitSystemTests.csproj

```diff
@@ -47,6 +47,10 @@
       <HintPath>$(PACKAGESPATH)\DynamoVisualProgramming.WpfUILibrary\lib\net48\DynamoCoreWpf.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="DynamoUtilities">
+      <HintPath>$(PACKAGESPATH)\DynamoVisualProgramming.Core\lib\net48\DynamoUtilities.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
```

Comment:
Why upgrading version requires the dependency change?

Reply:
![image](https://user-images.githubusercontent.com/62596335/175262456-328572f9-56da-4168-9e5c-a27ceef85661.png)